pkg/x/vmwscript/examples/swarm.json

```diff
@@ -0,0 +1,79 @@
+{
+    "label":"Docker-CE-on-CentOS",
+    "version":"0.1",
+    "vmconfig" : {
+        "datacentre" : "",
+        "datastore":"",
+        "network" : "",
+        "host" : "",
+        "guestCredentials" : {
+            "guestUser" : "root",
+            "guestPass" :"password"
+        }
+    },
+    "deployment": [
+        {"name": "Swarm Manager",
+            "note": "Build Swarm leader from Template",
+            "task":{
+               "inputTemplate": "DockerTemplate",
+               "outputName": "manager001",
+               "outputType": "VM",
+               "import":"",
+               "commands": [
+                   {
+                       "type":"execute",                    
```

Comment:
Are there types other than `execute`?  I guess a README would be helpful in understanding the design and the schema.  I take it that this executed by the govmomi library?  How does it execute this code on the guest -- does it require SSH or is that through some trusted hypervisor code?

I'd like to understand this more.... maybe we could even leverage some of the patterns here and have a nice integration with the template engine and other parts of infrakit.